### PR TITLE
Create thirdweb.md

### PR DESCRIPTION
**Pull request by fabriguespe**
Thirdweb blog post

### CODE REVIEW

File: blog/thirdweb.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
+---
+slug: thirdbweb-wallet-upload
+hide_table_of_contents: true
+title: "Introducing Wallet SDK and Sending remote storage"
+date: 2023-05-12
+authors: eng
```
Comment (fabriguespe) [inline blog/thirdweb.md:6]: fabri

File: blog/thirdweb.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+<!--truncate-->
+
+## Concepts
+
+### Thirdweb WalletSDK
+The WalletSDK is a development kit that grants developers access to a comprehensive selection of wallets, ranging from custodial to MPC to smart contracts.
+[Read more](https://twitter.com/thirdweb/status/1654191962751389697)
+
+### XMTP content types
```
Comment (jhaaaa) [inline blog/thirdweb.md:30]: We follow the Google developer doc style guide for headings - so sentence case and imperative mood for tasks: https://developers.google.com/style/headings. I've updated the heading according to this style guidance. =)

File: blog/thirdweb.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
+---
+slug: thirdweb-wallet-upload
+hide_table_of_contents: false
+title: "Introducing Wallet SDK and Sending remote storage"
+date: 2023-05-17
+authors: fabri
+image: "https://blog.thirdweb.com/content/images/size/w2000/2023/05/How-to-use-smart-wallet-.png"
+description: "Sending remote attachments with XMTP"
```
Comment (jhaaaa) [inline blog/thirdweb.md:8]: I think we can update this to the introduction text: Creating an effective 'Connect Wallet' flow is a challenging aspect of Web3 development. Thankfully, the team at Thirdweb has developed an excellent SDK, simplifying this process and enabling a superior wallet experience for your users.

File: blog/thirdweb.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+tags:
+- Content Types
+- SDKs
+- Media
+- Developers
+- WalletSDK
+- Thirdweb
+---
+
+Creating an effective 'Connect Wallet' flow is a challenging aspect of Web3 development. Thankfully, the team at Thirdweb has developed an excellent SDK, simplifying this process and enabling a superior wallet experience for your users.
```
Comment (jhaaaa) [inline blog/thirdweb.md:18]: Is this Thirdweb or thirdweb?

File: blog/thirdweb.md [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+tags:
+- Content Types
+- SDKs
+- Media
+- Developers
+- WalletSDK
+- Thirdweb
+---
+
+Creating an effective 'Connect Wallet' flow is a challenging aspect of web3 development. Thankfully, the team at Thirdweb has developed an excellent SDK, simplifying this process and enabling a superior wallet experience for your users.
```
Comment (jhaaaa) [inline blog/thirdweb.md:18]: Style nit: Changed "Web3" to "web3" - we use this format at XMTP Labs.

File: blog/thirdweb.md [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+
+<!--do we want to request a blog card visual from Nad?-->
+
+<!--truncate-->
```
Comment (jhaaaa) [inline blog/thirdweb.md:22]: We insert this comment to tell docusaurus where to stop displaying content on the blog preview page: https://junk-range-possible-git-thirdweb-blog-post-xmtp-labs.vercel.app/blog

**Pull request closed without merging**